setup.cfg

```diff
@@ -1,2 +1,51 @@
+[metadata]
+name = fiberassign
+version = attr: fiberassign.__version__
+author = DESI Collaboration
+author_email = desi-data@desi.lbl.gov
+license = BSD-3-Clause
+license_files = LICENSE.rst
+url = https://github.com/desihub/specex
+description = DESI Fiber Assignment Tools
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+edit_on_github = True
+github_project = desihub/fiberassign
+; classifiers =
+;     Development Status :: 5 - Production/Stable
+;     Environment :: Console
+;     Intended Audience :: Science/Research
+;     Operating System :: OS Independent
+;     Programming Language :: Python :: 3
+;     Topic :: Scientific/Engineering :: Astronomy
+;     Topic :: System :: Software Distribution
+;     Topic :: Utilities
+
+[options]
+zip_safe = False
+package_dir =
+    =py
+# find_namespace avoids a warning about data directories when building packages.
+packages = find_namespace:
+include_package_data = True
+python_requires = >=3.10
+# setup_requires = setuptools_scm
+install_requires =
+    numpy
+    pyyaml
+    scipy
+    matplotlib
+    astropy
+    fitsio
```

Comment:
fiberassign also uses desiutil (on PyPI) and desitarget (not yet on PyPI).

Reply:
Agreed. Will do.